cmd/nvidia-dra-controller/imex.go

```diff
@@ -259,3 +290,25 @@ func generateImexChannelPool(imexDomain string, numChannels int) resourceslice.P
 
 	return pool
 }
+
+// cleanupImexChannels removes all resource slices created by the IMEX manager.
+func (m *ImexManager) cleanupImexChannels() error {
```

Comment:
can we call this function `cleanupResourceSlices` to be symmetrical with `manageResourceSlices`

Reply:
done